GameServerLib/Logic/Packets/PacketHandlers/Handlers/HandleClick.cs

```diff
@@ -1,23 +1,31 @@
 ﻿using ENet;
-using LeagueSandbox.GameServer.Logic.Packets;
+using LeagueSandbox.GameServer.Core.Logic;
+using LeagueSandbox.GameServer.Core.Logic.PacketHandlers;
 using LeagueSandbox.GameServer.Logic.Players;
 
-namespace LeagueSandbox.GameServer.Core.Logic.PacketHandlers.Packets
+namespace LeagueSandbox.GameServer.Logic.Packets.PacketHandlers.Handlers
 {
-    class HandleClick : IPacketHandler
+    [Disabled]
```

Comment:
Could this be renamed to `DisabledPacket` or `PacketDisabled`, or somehow clarify this attribute __only__ works for packets?